source/lib-intro.tex

```diff
@@ -430,7 +452,7 @@
 \pnum
 Requirements are stated in terms of well-defined expressions that define valid terms of
 the types that satisfy the requirements. For every set of well-defined expression
-requirements there is a table that specifies an initial set of the valid expressions and
+requirements there is either a named concept or a table that specifies an initial set of the valid expressions and
```

Comment:
As explained elsewhere, this seems to be a lie for Cpp98ForwardIterator (for instance); the concept is not so named in the subclause, and the table of that name just shows syntax, not semantics.

Reply:
The tables typically have both syntactic and semantic requirements, but fail to include additional semantic requirements specified in prose in the section that contains the table. Do you agree that retitling the sections will clarify the intent?